### DISCUSSION TITLE
Implement list component via reanimated

### DISCUSSION THREAD

**Discussion by ex3ndr** [software-mansion/react-native-reanimated#2187] [Ideas]
For a while React Native community struggled with FlatList performance. One of the biggest issues of FlatList is that it must measure all elements and layout them correctly. This causes a lot of js-native interactions that shouldn't be needed.

Since `measure()` exists it could be possible to implement performant and stable FlatList where all computations are done on layout thread without needing to jump back to react.

But it seems that hook model is quite limited to fixed amount of views and should be expanded (how?) to support arbitrary number of views.

**Comment by Noitidart** [comment]
It looks like someone is developing a library for this soon - https://twitter.com/mrousavy/status/1472863750952792065

It also seems better than FlatList is RecyclerListView
